api-style-guide.md

```diff
@@ -883,18 +884,18 @@ For more specific info on the query parameter usage, see [URI Standards](#uri-st
 
 The data model for the representation MUST conform to [JSON][30]. The values may themselves be objects, strings, numbers, booleans, or arrays of objects.
 
-- Key names MUST camel cased words.
+- Key names MUST camelCased words.
   - `foo`
   - `barBaz`
 - Prefix such as `is` or `has` SHOULD NOT be used for keys of type boolean.
 - Fields that represent arrays SHOULD be named using plural nouns (e.g. authenticators-contains one or more authenticators, products-contains one or more products).
 
 <h2 id="enum-names">Enum Names</h2>
 
-Entries (values) of an enum SHOULD be composed of only upper-case alphanumeric characters and the underscore character, ( \_ ).
+Entries (values) of an enum SHOULD be composed of only upper and lower case alphanumeric characters.
 
-- `FIELD_10`
-- `NOT_EQUAL`
+- `inProgress`
```

Comment:
I think here we can stick to `FIELD_10`, that's a common thing for enums right?

Reply:
yeah, but the enums in the api v2 yaml file seem to use a mix of camel case and all uppercase, does that mean that they should be updated as well?